lib/main.dart

```diff
@@ -23,6 +24,11 @@ void main() async {
   final prefs = await SharedPreferences.getInstance();
   final showOnboarding = prefs.getBool('showOnboarding') ?? true;
 
+  SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
```

Comment:
This should be resolved later when my custom theme ticket gets merged. I am still waiting for someone to review it tho...

Reply:
ok that's good
So should I close the PR or leave it here for now?

Reply:
change it to draft PR for now please :)